python/cudf/tests/test_replace.py

```diff
@@ -84,23 +86,31 @@ def test_dataframe_replace():
 @pytest.mark.parametrize(
     'inplace',
     [True, False])
-def test_series_fillna_numerical(dtype, fill_type, null_value, inplace):
-    data = np.array([0, 1, null_value, 2, null_value], dtype='float64')
-    sr = Series(data).astype(dtype)
+def test_series_fillna_numerical(data_dtype, fill_dtype,
+                                 fill_type, null_value, inplace):
+    # TODO: These tests should use Pandas' nullable int type
+    # when we support a recent enough version of Pandas
+    # https://pandas.pydata.org/pandas-docs/stable/user_guide/integer_na.html
```

Comment:
We support the most recent version of Pandas AFAIK, but given most users aren't using the special Nullable types yet I don't think this is super high priority.